db/bigtable_eth1.go

```diff
@@ -2178,8 +2184,10 @@ func (bigtable *Bigtable) GetInternalTransfersForTransaction(transaction []byte,
 
 	names := make(map[string]string)
 	for _, t := range transfers {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
+		if t != nil {
```

Comment:
This should not suffice as the same `transfers` are accessed here anyways again:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L2210